test/test_transforms.py

```diff
@@ -1315,6 +1219,115 @@ def test_random_erasing(self):
         t.__repr__()
 
 
+def test_randomresized_params():
+    height = random.randint(24, 32) * 2
+    width = random.randint(24, 32) * 2
+    img = torch.ones(3, height, width)
+    to_pil_image = transforms.ToPILImage()
+    img = to_pil_image(img)
+    size = 100
+    epsilon = 0.05
+    min_scale = 0.25
+    for _ in range(10):
+        scale_min = max(round(random.random(), 2), min_scale)
+        scale_range = (scale_min, scale_min + round(random.random(), 2))
+        aspect_min = max(round(random.random(), 2), epsilon)
+        aspect_ratio_range = (aspect_min, aspect_min + round(random.random(), 2))
+        randresizecrop = transforms.RandomResizedCrop(size, scale_range, aspect_ratio_range)
+        i, j, h, w = randresizecrop.get_params(img, scale_range, aspect_ratio_range)
+        aspect_ratio_obtained = w / h
+        assert((min(aspect_ratio_range) - epsilon <= aspect_ratio_obtained and
+                aspect_ratio_obtained <= max(aspect_ratio_range) + epsilon) or
+               aspect_ratio_obtained == 1.0)
+        assert isinstance(i, int)
+        assert isinstance(j, int)
+        assert isinstance(h, int)
+        assert isinstance(w, int)
+
+
+@pytest.mark.parametrize('height, width', [
+    # height, width
+    # square image
+    (28, 28),
+    (27, 27),
+    # rectangular image: h < w
+    (28, 34),
+    (29, 35),
+    # rectangular image: h > w
+    (34, 28),
+    (35, 29),
+])
+@pytest.mark.parametrize('osize', [
+    # single integer
+    22, 27, 28, 36,
+    # single integer in tuple/list
+    [22, ], (27, ),
+])
+@pytest.mark.parametrize('max_size', (None, 37, 1000))
+def test_resize(height, width, osize, max_size):
+    img = Image.new("RGB", size=(width, height), color=127)
+
+    t = transforms.Resize(osize, max_size=max_size)
+    result = t(img)
+
+    msg = "{}, {} - {} - {}".format(height, width, osize, max_size)
```

Comment:
Just for info: we don't need these kind of error message anymore as they just print the compared values. Now that we're using pytest, we can just rely on pytest introspection. It's fine to leave as-is though ;)

Reply:
Awesome! I'll keep this in mind with my future PRs.